deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -642,7 +647,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.labels['name']
                 - name: DEBUG
-                  value: "false"
+                  value: "true"
```

Comment:
Same here.

Reply:
sorry I mistakenly commit `operator.yaml`. I had reverted it now